internal/cli/config/config_describe.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/spf13/cobra"
+)
+
+type ListOpts struct {
+	name  string
+	store config.Config
+}
+
+func (opts *ListOpts) Run() error {
+	config.SetName(&opts.name)
+	configDescription, err := config.GetConfigDescription()
+	if err != nil {
+		return err
+	}
+
+	for k, v := range configDescription {
+		fmt.Printf("%v = %v\n", k, v)
+	}
+
+	return nil
+}
+
+func DescribeBuilder() *cobra.Command {
+	opts := &ListOpts{
+		store: config.Default(),
```

Comment:
you don't read from the store but from global so similar to above you can move to use the store (good for testing) or remove it